pkg/webui/console/views/admin-packet-broker/messages.js

```diff
@@ -64,4 +64,8 @@ export default defineMessages({
     'Are you sure you want to deregister your network from Packet Broker?{lineBreak}' +
     'This will <b>permanently delete</b> all routing policies and may stop traffic from flowing.{lineBreak}' +
     'Traffic may still be forwarded to your network based on default routing policies configured by forwarders.',
+  defaultGatewayVisibility: 'Default gateway visibility',
+  gatewayVisibilityInformation:
+    'You can use the checkboxes below to control the default gateway visibility configuration.',
```

Comment:
```suggestion
    'You can use the checkboxes below to control what information your gateways will expose when interacting with other registered networks.',
```
@johanstokking pls ACK

Reply:
Maybe "You can use the checkboxes to control what information of your gateways will be visible"

This is a _default_ visibility, meaning that it's visible to anyone, even the public (so not only registered networks)